hw/ip/otbn/README.md

```diff
@@ -0,0 +1,41 @@
+# OpenTitan Big Number Accelerator (OTBN)
+
+This directory contains the implementation of the OpenTitan Big Number
+Accelerator (OTBN). OTBN is a coprocessor for asymmetric cryptographic
+operations like RSA or Elliptic Curve Cryptography (ECC).
+
+See https://docs.opentitan.org/hw/ip/otbn/doc/index.html for documentation on
+the current version of OTBN; documentation matching the code in this directory
+can be found in the `doc` directory.
+
+OTBN is currently in early development. Please ask questions and report issues
+through the [GitHub issue tracker](https://github.com/lowRISC/opentitan/issues).
+
+## Develop OTBN
+
+## Build the RTL implementation
+
+To build the RTL implementation of OTBN, do *not* pass the `OTBN_MODEL` flag to
+`fusesoc` and compile the simulation as usual. For example, the Verilator
```

Comment:
I think this could probably be phrased more clearly, since it looks a little bit like someone might "pass the OTBN_MODEL flag and compile the simulation as usual" and we're telling them not to. Maybe:

```
To build the RTL implementation of OTBN in a simulation, run `fusesoc` without passing the
`OTBN_MODEL` flag. For example, the Verilator...
```